alacritty_terminal/src/term/mod.rs

```diff
@@ -1317,6 +1302,12 @@ impl<T> TermInfo for Term<T> {
     }
 }
 
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum ClipboardType {
+    Clipboard,
+    Selection,
+}
```

Comment:
Right now this is inbetween an `impl Term` and `impl XXX for Term`, right? Might be better to put it after the handler impl so it's not in between two impls for the same thing?